docs/UserGuide.md

```diff
@@ -94,22 +94,20 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing an internship entry : `edit`
 
-Edits an existing person in the address book.
+Edits an internship entry from the list of existing entries.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [r/ROLE] [s/STATUS ] [d/DATE]`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+* The internship entry whose entry number is `INDEX` would be updated. `INDEX` needs to be a valid entry number as specified in the internship list displayed using `list` command. 
+* Items in square brackets are optional, but at least one of the optional fields must be provided.
```

Comment:
I am not sure whether we need to include the line "Items in square brackets are optional". We included in the Google docs README because we did not have a _Notes about the command format_ section. Do you think it is better if we mention what do thesquare brackets do in the _Notes about the command format section_ instead?

Reply:
Okie

Reply:
I placed this under the Notes about the command format section. I think we can just remove it.